src/scripts.js

```diff
@@ -16,7 +16,7 @@ import hydrationData from "../data/hydration";
 
 // An example of how you tell webpack to use a CSS (SCSS) file
 import './css/normalize.css';
-import './css/styles.css';
+import './css/styles.scss';
```

Comment:
They also wanted an .scss variable file, should we make one of those as well?  Or does styles.scss import the variables?